frontend/src/pages/PipelineDetailsTest.test.tsx

```diff
@@ -31,6 +31,12 @@ import * as StaticGraphParser from '../lib/StaticGraphParser';
 import { PageProps } from './Page';
 import PipelineDetails from './PipelineDetails';
 import * as WorkflowUtils from 'src/lib/v2/WorkflowUtils';
+import jsyaml from 'js-yaml';
+import fs from 'fs';
+import YAML from 'yaml';
```

Comment:
Do we use this library?

Reply:
We use fs only. Will delete unused parts.